storybookjs/storybook
# Update good first issue link to use correct Github label

Issue: Contribution guide linked to non-existent label on Github.

## What I did
Updated link to reference the "good first issue" label rather than the non-existent "good first contribution" label.

## How to test
1. Run documentation server
2. Navigate to `/basics/community/`
3. Verify that the link in the third item links to the issues on Github, filtered for issues labeled "good first issue" 

Is this testable with jest or storyshots?
N/A

Does this need a new example in the kitchen sink apps?
N/A

Does this need an update to the documentation?
Yes

If your answer is yes to any of these, please make sure to include it in your PR.
